api/tests/opentrons/drivers/test_command_builder.py

```diff
@@ -0,0 +1,78 @@
+from opentrons.drivers.command_builder import CommandBuilder
+
+
+def test_builder_create_command_with_terminator() -> None:
+    """It should create a command with terminator."""
+    terminator = "terminator"
+    builder = CommandBuilder(terminator=terminator)
+    assert builder.build() == "terminator"
+
+
+def test_builder_create_command_with_float() -> None:
+    """It should create a command with a floating point value."""
+    terminator = "terminator"
+    builder = CommandBuilder(terminator=terminator)
+    assert builder.with_float(
+        prefix='Z', value=1.2342, precision=3
+    ).build() == "Z1.234 terminator"
+
+
+def test_builder_create_command_with_float_no_round() -> None:
+    """It should create a command with a floating point value that is
+    not rounded."""
+    terminator = "terminator"
+    builder = CommandBuilder(terminator=terminator)
+    assert builder.with_float(
+        prefix='Z', value=1.23442, precision=None
+    ).build() == "Z1.23442 terminator"
+
+
+def test_builder_create_command_with_int() -> None:
+    """It should create a command with an integer point value."""
+    terminator = "terminator"
+    builder = CommandBuilder(terminator=terminator)
+    assert builder.with_int(
+        prefix='Z', value=15
+    ).build() == "Z15 terminator"
+
+
+def test_builder_create_command_with_gcode() -> None:
+    """It should create a command with a GCODE."""
+    terminator = "terminator"
+    builder = CommandBuilder(terminator=terminator)
+    assert builder.with_gcode(
+        gcode='G321'
+    ).build() == "G321 terminator"
+
+
+def test_builder_create_command_with_builder() -> None:
+    """It should create a command words in another builder."""
+    terminator = "terminator"
+    builder = CommandBuilder(terminator=terminator)
+    assert builder.with_gcode(
+        gcode='G321'
+    ).build() == "G321 terminator"
```

Comment:
General test style question. Should this be

```suggestion
    terminator = "terminator"
    builder = CommandBuilder(terminator=terminator)
    assert builder.with_gcode(
        gcode='G321'
    ).build() == f"G321 {terminator}"
```

or

```suggestion
    builder = CommandBuilder(terminator="terminator")
    assert builder.with_gcode(
        gcode='G321'
    ).build() == "G321 terminator"
```

?

In other words, should sentinel values be duplicated, or lifted into variables?

Reply:
I went ahead and made veriables. Thanks